onward/app/controllers/TechFeedbackController.scala

```diff
@@ -1,11 +1,53 @@
 package controllers
 
+import javax.inject.Inject
+
+import conf.Configuration
 import common._
 import model.Cached.RevalidatableResult
 import model.{ApplicationContext, Cached, MetaData, SectionSummary}
+import play.api.data.Form
+import play.api.libs.json.Json
 import play.api.mvc.{Action, Controller}
+import play.api.libs.ws._
+import play.api.data.Forms._
+import scala.concurrent.duration._
+
+class TechFeedbackController @Inject() (ws: WSClient) (implicit context: ApplicationContext) extends Controller with Logging {
```

Comment:
Can we not use `javax.inject` for dependency injection? The rest of the project use compile time DI with MacWire